app.js

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-unused-vars */
+/* eslint-disable linebreak-style */
```

Comment:
- [ OPTIONAL] You should not disable linters for they help you stick to best practice and have clean code base.